[2021-10-16 15:55:42] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8000 时出现错误: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 将重试. 
[2021-10-16 15:55:42] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":1845464277,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":24,"packet_sent":19,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1634370942}
 到 127.0.0.1:8000 失败: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 
[2021-10-16 15:55:47] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8000 时出现错误: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 将重试. 
[2021-10-16 15:55:47] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":1845464277,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":24,"packet_sent":19,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1634370947}
 到 127.0.0.1:8000 失败: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 
[2021-10-16 15:55:52] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8000 时出现错误: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 将重试. 
[2021-10-16 15:55:52] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":1845464277,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":24,"packet_sent":19,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1634370952}
 到 127.0.0.1:8000 失败: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 
[2021-10-16 15:55:57] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8000 时出现错误: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 将重试. 
[2021-10-16 15:55:57] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":1845464277,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":24,"packet_sent":19,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1634370957}
 到 127.0.0.1:8000 失败: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 
[2021-10-16 15:56:02] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8000 时出现错误: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 将重试. 
[2021-10-16 15:56:02] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":1845464277,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":24,"packet_sent":19,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1634370962}
 到 127.0.0.1:8000 失败: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 
[2021-10-16 15:56:07] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8000 时出现错误: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 将重试. 
[2021-10-16 15:56:07] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":1845464277,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":25,"packet_sent":20,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1634370967}
 到 127.0.0.1:8000 失败: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 
[2021-10-16 15:56:12] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8000 时出现错误: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 将重试. 
[2021-10-16 15:56:12] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":1845464277,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":25,"packet_sent":20,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1634370972}
 到 127.0.0.1:8000 失败: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 
[2021-10-16 15:56:17] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8000 时出现错误: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 将重试. 
[2021-10-16 15:56:17] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":1845464277,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":25,"packet_sent":20,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1634370977}
 到 127.0.0.1:8000 失败: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 
[2021-10-16 15:56:22] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8000 时出现错误: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 将重试. 
[2021-10-16 15:56:22] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":1845464277,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":25,"packet_sent":20,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1634370982}
 到 127.0.0.1:8000 失败: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 
[2021-10-16 15:56:27] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8000 时出现错误: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 将重试. 
[2021-10-16 15:56:27] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":1845464277,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":25,"packet_sent":20,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1634370987}
 到 127.0.0.1:8000 失败: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 
[2021-10-16 15:56:32] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8000 时出现错误: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 将重试. 
[2021-10-16 15:56:32] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":1845464277,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":25,"packet_sent":20,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1634370992}
 到 127.0.0.1:8000 失败: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 
[2021-10-16 15:56:37] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8000 时出现错误: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 将重试. 
[2021-10-16 15:56:37] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":1845464277,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":26,"packet_sent":21,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1634370997}
 到 127.0.0.1:8000 失败: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 
[2021-10-16 15:56:42] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8000 时出现错误: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 将重试. 
[2021-10-16 15:56:42] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":1845464277,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":26,"packet_sent":21,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1634371002}
 到 127.0.0.1:8000 失败: Post "http://127.0.0.1:8000": dial tcp 127.0.0.1:8000: connect: connection refused 
